### DISCUSSION TITLE
Check out the JSBSim Online Reference Manual (JORM)

### DISCUSSION THREAD

**Discussion by agodemar** [JSBSim-Team/jsbsim#386] [Show and tell]
**Heads up**

The sources are here: https://github.com/JSBSim-Team/jsbsim-reference-manual

The rendered website is here: https://jsbsim-team.github.io/jsbsim-reference-manual

**Comment by bcoconni** [comment]
Moved the issue #8 to `Discussions` and pinned it so that it can hopefully receive more attention.

**Comment by beatsal** [comment]
Not sure how good the ref manual is. Trying to d'load JSBSim but no luck. Can't even find the jsb.exe file! kendias@hotmail.com

**Reply by agodemar** [reply]
@beatsal Not sure you've actually gone through the materials. You don't need luck, you need to read.

In any case, here's the release page: [https://github.com/JSBSim-Team/jsbsim/releases](https://github.com/JSBSim-Team/jsbsim/releases) where you'll find installers for Windows, Ubuntu/Linux, and Python.

**Comment by 0x62** [comment]
How up to date is the PDF manual? The copyright date is 2011, but I'm not sure if that's the last revision (and if so, whether there have been substantial changes to the config schema since then?).

I'm currently working on a web-based reimplementation of JSBSimCommander, but I don't have a huge amount of experience with JSBSim so the more info the better. 

https://jsbsim.sourceforge.net/JSBSimReferenceManual.pdf

**Reply by agodemar** [reply]
> How up to date is the PDF manual? The copyright date is 2011, but I'm not sure if that's the last revision (and if so, whether there have been substantial changes to the config schema since then?).
> 
> I'm currently working on a web-based reimplementation of JSBSimCommander, but I don't have a huge amount of experience with JSBSim so the more info the better.
> 
> https://jsbsim.sourceforge.net/JSBSimReferenceManual.pdf

@0x62 thanks for the enquiry. I've seen your project [jsbsim-wasm](https://github.com/0x62/jsbsim-wasm) and it looks very interesting. 

The [jsbsim-team.github.io/jsbsim-reference-manual](https://jsbsim-team.github.io/jsbsim-reference-manual/) is newer than the [JSBSimReferenceManual.pdf](https://jsbsim.sourceforge.net/JSBSimReferenceManual.pdf) manual. The former is an ongoing, dynamic documentation project, while the latter is a frozen PDF marked 2011.

Yet, as far as I know, the XML specs did not change since then. Maybe @bcoconni and @seanmcleod70 have more on this.

**Comment by beatsal** [comment]
Thanks
________________________________

**Comment by seanmcleod70** [comment]
>  (and if so, whether there have been substantial changes to the config schema since then?).

Pretty sure there haven't been substantial changes, but there have been some changes.

For the most up to date documentation on the schema I suggest you take a look at the following in the repo.

https://github.com/JSBSim-Team/jsbsim/blob/master/JSBSim.xsd

https://github.com/JSBSim-Team/jsbsim/blob/master/JSBSimCommon.xsd

**Reply by 0x62** [reply]
> For the most up to date documentation on the schema I suggest you take a look at the following in the repo.

Thanks, I've been using this as well, and referencing it against background context in the PDF.
